### ISSUE TITLE
Ability to add/remove albums from Spotify

### ISSUE THREAD

**Issue by maxsubs** [vhaudiquet/BladePlayer#17]
In the same way we can already add/remove songs or playlists from our Spotify playlists. This way we don't have to use the official app for this.

**Comment by vhaudiquet** [comment]
The issue with that can be API rate limits from Spotify.
I removed some of those features (that were available in v1) because i kept getting rate-limited by Spotify.

The difference is that now i have a request quota extension.

Maybe it is possible, i would need to benchmark how many API calls are needed.

**Comment by maxsubs** [comment]
hey! great to see you! I hope we'll see an update soon, I've seen you've checked some of the issues as completed, does it mean a new version is around the corner? Thanks anyhow!

**Comment by vhaudiquet** [comment]
Yes, i'm about to release a version 2.2, with new features and hopefully less bugs

I have been working on many other work and/or personal projects so i had no time left for Blade, but i'm motivated again to add new features

I'll at least try to fix all the issues present in the repo :)

**Comment by TomBonnot** [comment]
I don't want to open a new ticket for this so I am writing on a lookalike topic :

First of all, this is a beautiful work you are doing on this app and I cheer you up for this. 

As for the topic, it was a suggestion enhancement to add a "Liked" button for Spotify. As you said before, if it's not here actually it is mostly because you get rate limited by Spotify; so I guess that it will be added later, but you need time. 

So I am just adding this comment to make a following "ticket" of this, but I know that you have way more to do before adding this haha. 

Again, gg for your work !
